crates/vsock/src/vhu_vsock_thread.rs

```diff
@@ -174,26 +174,40 @@ impl VhostUserVsockThread {
             }
             break 'epoll;
         }
+
+        self.mem.is_some()
```

Comment:
It seems like we are using the presence of a memory map as a side effect to detect we are ready to go. Is this always going to be the case? I'm thinking of cases where we might be dynamically mapping memory as we go.

Where does self.mem get set anyway?